libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ConfusionMatrixHeatmap.tsx

```diff
@@ -0,0 +1,311 @@
+        })
+      );
+      selectedLabels.push(...confusionMatrixData.selectedLabels);
+    }
+
+    const theme = getTheme();
+
+    let classSelectionErrorMessage: string | undefined;
+    if (this.state.selectedClasses.length < this.minDisplayableClasses) {
+      classSelectionErrorMessage = localization.formatString(
+        localization.ModelAssessment.ModelOverview.confusionMatrix
+          .confusionMatrixClassMinSelectionError,
+        this.minDisplayableClasses
+      );
+    } else if (this.state.selectedClasses.length > this.maxDisplayableClasses) {
+      classSelectionErrorMessage = localization.formatString(
+        localization.ModelAssessment.ModelOverview.confusionMatrix
+          .confusionMatrixClassMaxSelectionError,
+        this.maxDisplayableClasses
+      );
+    }
+
+    return (
+      <Stack id="modelOverviewConfusionMatrix">
+        <Stack horizontal>
+          <StackItem className={classNames.dropdown}>
+            <ComboBox
+              id="confusionMatrixCohortDropdown"
+              label={
+                localization.ModelAssessment.ModelOverview.confusionMatrix
+                  .confusionMatrixCohortSelectionLabel
+              }
+              selectedKey={selectedCohort.cohort.getCohortID()}
+              options={this.context.errorCohorts.map(
+                (errorCohort: ErrorCohort) => {
+                  return {
+                    key: errorCohort.cohort.getCohortID(),
+                    text: errorCohort.cohort.name
+                  };
+                }
+              )}
+              onChange={this.onSelectCohort}
+              styles={FluentUIStyles.limitedSizeMenuDropdown}
+            />
+          </StackItem>
+          <StackItem className={classNames.dropdown}>
+            <ComboBox
+              id="confusionMatrixClassDropdown"
+              placeholder={
+                localization.ModelAssessment.ModelOverview.confusionMatrix
+                  .confusionMatrixClassSelectionDefaultPlaceholder
+              }
+              label={
+                localization.ModelAssessment.ModelOverview.confusionMatrix
+                  .confusionMatrixClassSelectionLabel
+              }
+              selectedKey={this.state.selectedClasses}
+              options={this.state.allClasses.map((category: string) => {
+                return { key: category, text: category };
+              })}
+              errorMessage={classSelectionErrorMessage}
+              onChange={this.onSelectClasses}
+              multiSelect
+              styles={FluentUIStyles.limitedSizeMenuDropdown}
+            />
+          </StackItem>
+        </Stack>
+        {this.state.selectedClasses.length >= this.minDisplayableClasses &&
+          this.state.selectedClasses.length <= this.maxDisplayableClasses && (
+            <StackItem className={classNames.chart}>
+              <HeatmapHighChart
+                id="ModelOverviewConfusionMatrix"
+                configOverride={{
```

Comment:
looks like there is some issue with the global eslint rule. This file is over 300 lines but there is no lint error in the test. Can we move this configOverride in a separate file to reduce lines? I remember previously the max-line rule in eslintrc file in each folder works and pops up errors if any.

Reply:
We exempted the entirety of `model-assessment` so it's a consequence of that. I'll create a PR to undo that and exempt just specific files, so at least for new files it won't apply! I'll also reduce this file's size.